### PR TITLE
Adds support for teler

### PR DESCRIPTION
**Pull request by iosifache**
# Metadata

- **Fixed Issue**: #3 
- **Contributors**: @iosifache

# Proposed Changes

- Adding support for teler
- Adding a new parameter in the configuration setting callback for the old value of the aspect

# New Functioning

MutableSecurity now supports teler.

### CODE REVIEW

File: mutablesecurity/modules/solutions_manager/solutions/teler.py [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+            Teler.result = True
+
+            return
+
+        # If the version on GitHub is greater, then update the software
+        Teler.uninstall(state=state, host=host)
+        Teler.install(state=state, host=host)
+
+    @deploy
+    def uninstall(state, host):
```
Comment (AntociAlin) [inline mutablesecurity/modules/solutions_manager/solutions/teler.py:413]: What about the log files and the whole folder from `/opt/mutablesecurity/teler`? Shouldn't they also be deleted?

**Review by AntociAlin — CHANGES_REQUESTED** [review]
Almost everything seems fine and I tested everything. Check also the comment please.

File: mutablesecurity/modules/solutions_manager/solutions/teler.py (hunk 1 above)
Comment (iosifache) [reply mutablesecurity/modules/solutions_manager/solutions/teler.py:413]: Let's preserve the configuration files that we create for a specific solution (here `/opt/mutablesecurity/teler`). You may want to reinstall teler and keep the same configuration as in the previous installation. But yes, the log file that is set in the configuration (defaults to `/var/log/teler.log`) needs to be deleted.

Comment (iosifache) [reply mutablesecurity/modules/solutions_manager/solutions/teler.py:413]: @AntociAlin, I adapted the uninstallation operation. Please recheck it.

**Review by AntociAlin — APPROVED** [review]
Looks good. Well done.

**Pull request closed without merging**